website/docs/addon_aftereffects_artist.md

```diff
@@ -20,16 +22,48 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 -   [Publish](artist_tools_publisher)
 -   [Manage](artist_tools_inventory)
 
-## Setup
+## AE Extension installation
 
-To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. Then go to `{path to pype}hosts/aftereffects/api/extension.zxp`. 
+:::info
+Once your admin has activated `After Effects` in the production bundle, you need to run the AYON launcher at least once. This allows the launcher to download the addon to the correct location on your computer. 
+After that, you can continue with the installation instructions for the After Effects extension.
+:::
+
+<Tabs
+groupId="platforms"
+defaultValue="win"
+values={[
+{label: 'Windows', value: 'win'},
+{label: 'Linux', value: 'linux'},
+{label: 'Mac', value: 'mac'},
+]}>
+
+<TabItem value="win">
+
+1. Use your favorite app to install extensions. we recommend using [Anastasyi's Extension Manager](https://install.anastasiy.com/).
+2. Locate and install the `extension.zxp` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.
+
+:::tip
+You can quickly navigate to the local app data using the environment variable shortcut: `%LOCALAPPDATA%`. For example, using `%LOCALAPPDATA%\Ynput\AYON\addons` as path to see your installed addons.
+:::
+
+</TabItem>
+
+<TabItem value="linux">
+
+1. Use your favorite app to install extensions.
```

Comment:
Interesting - I can't find anything about installing adobe extensions on Linux. Are adobe products even supported on Linux?

Reply:
Quoted: 
```
= Can I run Photoshop on Ubuntu?
Adobe Photoshop is not natively available for Ubuntu. 
However, it can be installed on Ubuntu using the Wine compatibility layer. 
What is Wine? Wine is a compatibility layer that allows users to run Windows applications 
on Linux, macOS, and other Unix-like operating systems.Apr 13, 2022
```

Reply:
If so, we may actually not 'hint' at using their favorite app to install it and just say it's e.g. not officially supported on Linux? Or?

Reply:
@m-u-r-p-h-y  @kalisp what do you think we should say about it ?

Reply:
If they are interested how to install AYON integration for AE on LInux, they must figured out somehow how to install it, so   this documentation should be still applicable.